PostHog/Replay/UIImage+Util.swift

```diff
@@ -11,14 +11,29 @@
 
     extension UIImage {
         func toBase64(_ compressionQuality: CGFloat = 0.3) -> String? {
-            let jpegData = jpegData(compressionQuality: compressionQuality)
-            let base64 = jpegData?.base64EncodedString()
+            toWebPBase64(compressionQuality) ?? toJpegBase64(compressionQuality)
+        }
 
-            if let base64 = base64 {
-                return "data:image/jpeg;base64,\(base64)"
+        private func toWebPBase64(_ compressionQuality: CGFloat) -> String? {
```

Comment:
RN and Flutter will need to use this so it has to be a public method, unless we do the change via `imageToBase64` method

Reply:
all good then